account_fiscal_position_vat_check/tests/test_account_fiscal_position_vat_check.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountFiscalPositionVatCheck(TransactionCase):
+    def test_partner_must_have_vat_check(self):
+        fp = self.env['account.fiscal.position'].create({
+            'name': 'Test',
+            'customer_must_have_vat': True})
+        customer = self.env['res.partner'].search(
+            [('customer', '=', True)], limit=1)
+        customer.write({'property_account_position_id': fp.id})
+        res = customer.fiscal_position_change()
+        self.assertEqual(res['warning']['title'], 'Missing VAT number:')
+        fp.write({'customer_must_have_vat': False})
+        res = customer.fiscal_position_change()
+        self.assertEqual(res, None)
```

Comment:
`self.assertFalse(res)` better (because you can maybe return `{}` or `False` in other implementation)